### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by balatstar**
Checked page for accessibility. Applied fixes for the following:
- Page title: no issues found
- Image text alternatives (alt attributes): added missing
- Headings: updated hierarchy
- Contrast ratio: no issues found
- Resize text: no issues found
- Keyboard access: no issues found
- Forms, labels, errors: add for to labels
- Moving, Flashing, or Blinking Content: no issues found
- Multimedia (video, audio) alternatives: no issues found
- Basic structure check: no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
           <button class="cta-dark">Start collaboration</button>
         </div>
         <div class="hero-image">
-          <img src="images/image_geometry_2.svg" class="mobile-only" />
-          <img src="images/image-geometry_1.svg" class="desktop-only" />
+          <img src="images/image_geometry_2.svg" alt="" class="mobile-only" />
```
Comment (balatstar) [inline index.html:32]: Image text alternatives (alt attributes) - added alt text

File: index.html [hunk 2]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
+          <img src="images/image-geometry_1.svg" alt="" class="desktop-only" />
```
Comment (balatstar) [inline index.html:33]: Image text alternatives (alt attributes) - added alt text

File: index.html [hunk 3]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
               <ul class="projects-content-tags">
                 <li>HTML/CSS</li>
                 <li>Ruby on Rails</li>
                 <li>Javascript</li>
               </ul>
               <div class="projects-content-button">
-                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" /></button>
+                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" alt="See this project" /></button>
```
Comment (balatstar) [inline index.html:77]: Image text alternatives (alt attributes) - added alt text

File: index.html [hunk 4]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
```
Comment (balatstar) [inline index.html:91]: Image text alternatives (alt attributes) - added alt text

File: index.html [hunk 5]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
```
Comment (balatstar) [inline index.html:105]: Image text alternatives (alt attributes) - added alt text

File: index.html [hunk 6]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
```
Comment (balatstar) [inline index.html:119]: Image text alternatives (alt attributes) - added alt text

File: index.html [hunk 7]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-          <img src="images/image_geometry_2.svg" class="mobile-only" />
-          <img src="images/image-geometry_1.svg" class="desktop-only" />
+          <img src="images/image_geometry_2.svg" alt="" class="mobile-only" />
+          <img src="images/image-geometry_1.svg" alt="" class="desktop-only" />
         </div>
       </section>
       <section class="projects">
         <div class="projects-content">
-          <header class="proj-title"><h1>Projects</h1></header>
+          <header class="proj-title"><h2>Projects</h2></header>
```
Comment (balatstar) [inline index.html:38]: Headings: updated hierarchy

File: index.html [hunk 8]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
           <article class="projects-content-item project-0">
             <div class="projects-content-image"><img src="images/Rectangle21.jpg" alt="Project name" /></div>
             <div class="projects-content-group">
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
```
Comment (balatstar) [inline index.html:42]: Headings: updated hierarchy

File: index.html [hunk 9]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" /></button>
+                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" alt="See this project" /></button>
               </div>      
             </div>
           </article>
           <article class="projects-content-item project-1">
             <div class="projects-content-image"><img src="images/project1.png" alt="Project name" /></div>
             <div class="projects-content-group">
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
```
Comment (balatstar) [inline index.html:56]: Headings: updated hierarchy

File: index.html [hunk 10]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" /></button>
+                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" alt="See this project" /></button>
               </div>
             </div>
           </article>
           <article class="projects-content-item project-2">
             <div class="projects-content-image"><img src="images/project2.png" alt="Project name" /></div>
             <div class="projects-content-group">
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
```
Comment (balatstar) [inline index.html:70]: Headings: updated hierarchy

File: index.html [hunk 11]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" /></button>
+                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" alt="See this project" /></button>
               </div>
             </div>
           </article>
           <article class="projects-content-item project-3">
             <div class="projects-content-image"><img src="images/project3.png" alt="Project name" /></div>
             <div class="projects-content-group">
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
```
Comment (balatstar) [inline index.html:84]: Headings: updated hierarchy

File: index.html [hunk 12]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" /></button>
+                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" alt="See this project" /></button>
               </div>
             </div>
           </article>
           <article class="projects-content-item project-4">
             <div class="projects-content-image"><img src="images/project4.png" alt="Project name" /></div>
             <div class="projects-content-group">
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
```
Comment (balatstar) [inline index.html:98]: Headings: updated hierarchy

File: index.html [hunk 13]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" /></button>
+                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" alt="See this project" /></button>
               </div>
             </div>
           </article>
           <article class="projects-content-item project-5">
             <div class="projects-content-image"><img src="images/project5.png" alt="Project name" /></div>
             <div class="projects-content-group">
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
```
Comment (balatstar) [inline index.html:112]: Headings: updated hierarchy

File: index.html [hunk 14]
```diff
@@ -128,29 +128,29 @@ <h1>Charlie Giorgania</h1>
       </section>
       <section class="about">
         <div class="about-summary">
-          <h1>About</h1>
+          <h2>About</h2>
```
Comment (balatstar) [inline index.html:131]: Headings: updated hierarchy

File: index.html [hunk 15]
```diff
@@ -128,29 +128,29 @@ <h1>Charlie Giorgania</h1>
...
           <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
           <button class="about-summary-cta">Get my CV <img src="images/ic_download.svg" alt="Download" /></button>
         </div>
         <div class="about-list-main">
           <ul class="about-list-main2">
-            <li class="about-list languages"><h2 class="about-list-header">Languages</h2>
+            <li class="about-list languages"><h3 class="about-list-header">Languages</h3>
```
Comment (balatstar) [inline index.html:137]: Headings: updated hierarchy

File: index.html [hunk 16]
```diff
@@ -128,29 +128,29 @@ <h1>Charlie Giorgania</h1>
...
               <ul>
                 <li>JavaScript (ES5/ES6)</li>
                 <li>Ruby</li>
                 <li>HTML</li>
                 <li>CSS</li>
               </ul>
             </li>
-            <li class="about-list frameworks"><h2 class="about-list-header">Frameworks</h2>
+            <li class="about-list frameworks"><h3 class="about-list-header">Frameworks</h3>
```
Comment (balatstar) [inline index.html:145]: Headings: updated hierarchy

File: index.html [hunk 17]
```diff
@@ -128,29 +128,29 @@ <h1>Charlie Giorgania</h1>
...
               <ul>
                 <li>React.js</li>
                 <li>Ruby on rails</li>
                 <li>RSpec</li>
                 <li>Capybara</li>
               </ul>
             </li>
-            <li class="about-list skills"><h2 class="about-list-header">Skills</h2>
+            <li class="about-list skills"><h3 class="about-list-header">Skills</h3>
```
Comment (balatstar) [inline index.html:153]: Headings: updated hierarchy

File: index.html [hunk 18]
```diff
@@ -165,12 +165,12 @@ <h1>About</h1>
       <section class="contact-main">
         <div class="contact-main2">
           <div class="contact">
-            <h1>Get Started</h1>
+            <h2>Get Started</h2>
```
Comment (balatstar) [inline index.html:168]: Headings: updated hierarchy

File: index.html [hunk 19]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
                 <ul class="projects-content-tags">
                   <li>HTML/CSS</li>
                   <li>Ruby on Rails</li>
                   <li>Javascript</li>
                 </ul>
               <div class="projects-content-button">
-                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" /></button>
+                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" alt="See this project" /></button>
```
Comment (balatstar) [inline index.html:49]: Image text alternatives (alt attributes) - added alt text

File: index.html [hunk 20]
```diff
@@ -29,94 +29,94 @@ <h1>Charlie Giorgania</h1>
...
-              <div class="projects-content-name"><h2>Project name goes here</h2></div>
+              <div class="projects-content-name"><h3>Project name goes here</h3></div>
               <ul class="projects-content-tags">
                 <li>HTML/CSS</li>
                 <li>Ruby on Rails</li>
                 <li>Javascript</li>
               </ul>
               <div class="projects-content-button">
-                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" /></button>
+                <button class="cta-dark">See this project <img src="images/ic_arrow_right.svg" alt="See this project" /></button>
```
Comment (balatstar) [inline index.html:63]: Image text alternatives (alt attributes) - added alt text

File: index.html [hunk 21]
```diff
@@ -165,12 +165,12 @@ <h1>About</h1>
...
         <div class="contact-main2">
           <div class="contact">
-            <h1>Get Started</h1>
+            <h2>Get Started</h2>
             <p class="contact-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
             <form action="https://formspree.io/f/xpzegjzj" method="post" class="contact-form">
-              <label>Your name<input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required /></label>
-              <label class="contact-email">Your email<input type="email" id="email" name="email" placeholder="Email Address" required /></label>
-              <label>Your message<textarea name="text-box" id="message" name="message" cols="40" rows="6" maxlength="500" placeholder="Enter your message here..." required></textarea></label>
+              <label for="name">Your name<input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required /></label>
```
Comment (balatstar) [inline index.html:171]: Forms, labels, errors: add "for" to labels

File: index.html [hunk 22]
```diff
@@ -165,12 +165,12 @@ <h1>About</h1>
...
+              <label class="contact-email" for="email">Your email<input type="email" id="email" name="email" placeholder="Email Address" required /></label>
```
Comment (balatstar) [inline index.html:172]: Forms, labels, errors: add "for" to labels

File: index.html [hunk 23]
```diff
@@ -165,12 +165,12 @@ <h1>About</h1>
...
+              <label for="message">Your message<textarea name="text-box" id="message" name="message" cols="40" rows="6" maxlength="500" placeholder="Enter your message here..." required></textarea></label>
```
Comment (balatstar) [inline index.html:173]: Forms, labels, errors: add "for" to labels

**Review by balatstar — COMMENTED** [review]
Checked page for accessibility. 

- Page title: no issues found
- Contrast ratio: no issues found
- Resize text: no issues found
- Keyboard access: no issues found
- Moving, Flashing, or Blinking Content: no issues found
- Multimedia (video, audio) alternatives: no issues found
- Basic structure check: no issues found

Fixed issues:

**Comment by balatstar** [comment]
@LionRouge1 I was going to submit my previous comment for second review but the status of my project is already "SUBMIT". Not sure how to move forward... should I submit and merge now? Thank you!

**Comment by ahmetbozaci** [comment]
# 🥳 🎉🥇 💯  💥 🎆 🎈 STATUS: APPROVED 🥳 🎉  🥇  💯  💥   🎆 🎈 🎉 


@balatstar  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.
___ 

### 💯 Highlights 💯
- Well done show the line of the codes that you change according to the report.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**